.travis.yml

```diff
@@ -36,7 +52,7 @@ addons:
 
 install:
   - |
-    if [ $TRAVIS_PHP_VERSION != '5.6' ]; then
+    if [[ $TRAVIS_PHP_VERSION != '5.6' || $TRAVIS_PHP_VERSION != hhv* ]]; then
```

Comment:
this will disable xdebug always. should be `[ $TRAVIS_PHP_VERSION != '5.6' ] && [ $TRAVIS_PHP_VERSION != hhv* ]`